### DISCUSSION TITLE
How to use JSBSim UE on MacOS

### DISCUSSION THREAD

**Discussion by MiaFoxcat** [JSBSim-Team/jsbsim#791] [Support requests]
Hello!
I think the title is pretty self-explanatory I believe. Is there a way to use JSBSim for UE5 on MacOS?
Thanks in advance,
-Knedlik

**Comment by bcoconni** [comment]
@KnedlikMCPE,
Well, JSBSim standalone can be compiled and run successfully on MacOSX (which is one of the supported platform of JSBSim) and the same goes for Unreal Engine 5 (according to https://www.unrealengine.com/en-US/download) so I'd guess the answer is yes.

However as far as I know no one has reported using JSBSim for UE5 successfully on MacOSX so the [build procedure](https://github.com/JSBSim-Team/jsbsim/blob/master/UnrealEngine/README.md) might need some adjustments.

**Reply by MiaFoxcat** [reply]
I would use the standalone, are there any gotchas that would disallow me using it if I change the libJSBSim config to a shared library due to the license? I'm aware I cannot use the static one due to the license.

**Reply by bcoconni** [reply]
Not that I am aware of. JSBSim can be compiled as a shared library for that very reason: becoming compatible with the Unreal license.
You can find some further details about that topic in discussion #631.
